### PR TITLE
Make permissions explicit and precise

### PR DESCRIPTION
**Pull request by monaullah**
Inspired by @torbjokv's sharings:
- Make managed policies and define explicitly for lambdas. 
- Add policy `SecretsManagerPolicy` for `FetchNviInstitutionReportProxyHandler`

### CODE REVIEW

**Review by brinxmat — APPROVED** [review]

File: template.yaml [hunk 1]
```diff
@@ -327,6 +292,8 @@ Resources:
           - !Ref LambdaSubnet2
       Policies:
         - VPCAccessPolicy: { }
+        - !GetAtt NeptuneManagedPolicy.PolicyArn
+        - !GetAtt DeployToVpnPolicy.PolicyArn
       Handler: no.sikt.nva.data.report.api.fetch.FetchNviInstitutionReport::handleRequest
       Role: !GetAtt LambdaRole.Arn
```
Comment (torbjokv) [inline template.yaml:298]: fjerne denne?

File: template.yaml [hunk 2]
```diff
@@ -112,98 +112,62 @@ Resources:
...
+        Statement:
+          - Effect: Allow
+            Action:
+              - sqs:ReceiveMessage
+              - sqs:DeleteMessage
+              - sqs:SendMessage
+              - sqs:GetQueueAttributes
+            Resource:
+              - !GetAtt DataLoadingDLQ.Arn
+  DeployToVpnPolicy:
```
Comment (torbjokv) [inline template.yaml:157]: kan videre å erstatte denne med https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AWSLambdaVPCAccessExecutionRole.html kanskje?

Comment (monaullah) [reply template.yaml:157]: Ja, kan ta det i en ny PR

File: template.yaml [hunk 3]
```diff
@@ -112,98 +112,62 @@ Resources:
...
-    Properties:
-      PolicyName: !Sub DefaultLambdaPermissions-${AWS::StackName}
-      Roles:
-        - !Ref LambdaRole
+            Action:
+              - s3:Get*
+              - s3:List*
+              - s3:PutObject
+              - events:*
+            Resource: '*'
```
Comment (torbjokv) [inline template.yaml:130]: 😾

**Pull request merged**